templates/editCounter/general_stats.wikitext.twig

```diff
@@ -30,7 +30,7 @@
 {% endif %}
 |-
 | {{ msg('is-admin') }}
-| {% if user.isAdmin(project) %}1{% else %}0{% endif %}
+| {% if user.isAdmin(project) %}Yes{% else %}No{% endif %}
```

Comment:
This needs to be localized. I can show you how to do this if you'd like, but regardless I think this change should be saved for a separate PR. We could even show images for a green tick and red cross, like we do in the UI (but putting "yes" / "no" as the `alt` text, for accessibility).